features/bootstrap/FeatureContext.php

```diff
@@ -215,15 +219,15 @@ public function create_run_dir() {
 	public function build_phar( $version = 'same' ) {
 		$this->variables['PHAR_PATH'] = $this->variables['RUN_DIR'] . '/' . uniqid( "wp-cli-build-", TRUE ) . '.phar';
 
-		// Test running against WP-CLI proper
-		$make_phar_path = __DIR__ . '/../../utils/make-phar.php';
+		// Test running against a package installed as a WP-CLI dependency
+		// WP-CLI installed as a project dependency
+		$make_phar_path = __DIR__ . '/../../../../../utils/make-phar.php';
 		if ( ! file_exists( $make_phar_path ) ) {
-			// Test running against a package installed as a WP-CLI dependency
-			// WP-CLI installed as a project dependency
-			$make_phar_path = __DIR__ . '/../../../../../utils/make-phar.php';
+			// Test running against WP-CLI proper
+			$make_phar_path = __DIR__ . '/../../utils/make-phar.php';
 			if ( ! file_exists( $make_phar_path ) ) {
 				// WP-CLI as a dependency of this project
-				$make_phar_path = __DIR__ . '/../../vendor/wp-cli/wp-cli/utils/make-phar.php';
+				$make_phar_path = __DIR__ . '/../../vendor/wp-cli/phar-build/utils/make-phar.php';
```

Comment:
Isn't this going to break tests for packages because they don't have `wp-cli/phar-build` as a dependency?

I'm not convinced `make-phar.php` should be tracked in the `wp-cli/phar-build` repo, given some packages need to use it to build the Phar.